nixos/modules/services/monitoring/collectd.nix

```diff
@@ -43,6 +52,16 @@ in {
   options.services.collectd = with types; {
     enable = mkEnableOption "collectd agent";
 
+    validateConfig = mkOption {
+      default = true;
```

Comment:
I wonder if we should do `default = stdenv.hostPlatform == stdenv.buildPlatform;`? (And `defaultText = "stdenv.hostPlatform == stdenv.buildPlatform"`?). (Not important to me, and I don't know how other modules handle this.)

Reply:
I have had inconsistent reviews on similar PRs in the past, so let's leave it as is unless someone has a strong opinion.